api/src/pcapi/settings.py

```diff
@@ -133,8 +129,13 @@
 ADMINISTRATION_EMAIL_ADDRESS = os.environ.get("ADMINISTRATION_EMAIL_ADDRESS")
 COMPLIANCE_EMAIL_ADDRESS = os.environ.get("COMPLIANCE_EMAIL_ADDRESS", "")
 DEV_EMAIL_ADDRESS = os.environ.get("DEV_EMAIL_ADDRESS")
+
+""" Dans quels cas a t'on besoin de passer un `EMAIL_BACKEND` en  variable d'environnement ?
+Pour effectuer des tests de charge et éviter de polluer ou "encombrer"
+Sendinblue, il convient donc de surcharger `EMAIL_BACKEND` et de choisir
+le backend `pcapi.core.mails.backends.logger.LoggerBackend`."""
```

Comment:
(détails / doc) On a plutôt l'habitude de commenter en anglais. Suggestion : 

```suggestion
# When load testing, override `EMAIL_BACKEND` to avoid going over SendinBlue quota:
#     EMAIL_BACKEND="pcapi.core.mails.backends.logger.LoggerBackend"
```

(Ceci dit, je ne sais pas vraiment si ça a sa place ici. C'est déjà documenté dans [mails/CONTRIBUTING.md](https://github.com/pass-culture/pass-culture-main/tree/41b8cb7c0e2ca9e89a9aef84fcbbfc21e93be882/api/src/pcapi/core/mails). Mais on peut doubler l'information, pourquoi pas.)